JS-Algorithms/challenges.js

```diff
@@ -41,7 +56,13 @@ Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "German
 const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
-  // YOUR CODE HERE...
+  const CIRCULAR_INDEX = (index >= COUNTRY_NAMES.length) ? index - COUNTRY_NAMES.length : index;;
+
```

Comment:
If the result of the subtraction is greater than 5, the response will be the same array. To avoid this we can use `%` to get values from 0 to 4.